mica/archive/aca_dark/dark_cal.py

```diff
@@ -142,40 +144,70 @@ def get_dark_cal_id(date, select='before'):
 
 
 @DARK_CAL.cache
-def get_dark_cal_image(date, select='before', t_ccd_ref=None):
+def _get_dark_cal_image_props(date, select='before', t_ccd_ref=None, aca_image=False):
     """
-    Return the dark calibration image (e-/s) nearest to ``date``.
-
-    If ``select`` is ``'before'`` (default) then use the first calibration which
-    occurs before ``date``.  Other valid options are ``'after'`` and ``'nearest'``.
+    Return the dark calibration image (e-/s) nearest to ``date`` and the corresponding
```

Comment:
One of the `select` constraints *is* `nearest`, so this isn't perfect.  I understand, but if this dunder method has a bunch of stuff copied over from the public method in the docstring I don't know why I doesn't deserve the full treatment.